odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -273,7 +273,7 @@ Setting up a virtual machine
     :width: 600
 
 4. Click the **Advanced** tab at the top and copy and paste the contents from the :download:`cloud_init_AZURE.yml </files/cloud_init_AZURE.yml>` file into the *Custom data* box. Finally, click :guilabel:`Review + create` to actually create the machine.
-If you had generated the SSH key pair through azure automatic generate key pair option, then it now gives you a prompt to download the key (.pem) file. It is important to download it and remember the path to this file in your computer for connecting to virtual machine later.
+If you had generated the SSH key pair through AZURE automatic generate key pair option, then it now gives you a prompt to download the key (.pem) file. It is important to download it and remember the path to this file in your computer for connecting to virtual machine later.
```

Comment:
Sorry my last comment was not clear, I think it is "Azure" not "AZURE" or "azure"